### ISSUE TITLE
Issue with Batch Processing in predict_batch Function

### ISSUE THREAD

**Issue by codefanw** [yuwenmichael/Grounding-DINO-Batch-Inference#2]
The predict_batch function seems to only process the first image in a batch when generating predictions. Specifically, the lines:
```
prediction_logits = outputs["pred_logits"].cpu().sigmoid()[0]
prediction_boxes = outputs["pred_boxes"].cpu()[0]
```
These lines appear to only handle the logits and boxes for the first image in the batch, ignoring the rest.

**Comment by yuwenmichael** [comment]
Yes and that's the reason I create this repo, to solve the problem which only predicting an image a time. Pls check out [batch_utilities.py](https://github.com/yuwenmichael/Grounding-DINO-Batch-Inference/blob/main/batch_utlities.py) for batch inference.
